drivers/gles2/rasterizer_storage_gles2.h

```diff
@@ -60,21 +60,10 @@ class RasterizerStorageGLES2 : public RasterizerStorage {
 
 		bool shrink_textures_x2;
 		bool use_fast_texture_filter;
-		// bool use_anisotropic_filter;
-
-		bool hdr_supported;
-
-		bool use_rgba_2d_shadows;
-
-		// float anisotropic_level;
 
 		int max_texture_image_units;
 		int max_texture_size;
 
-		bool generate_wireframes;
```

Comment:
This should probably be supported I guess.. so i would leave this bool in there as a reminder?

Reply:
Re-added `generate_wireframes` with a todo tag